bridgesample/src/main/java/com/livefront/bridgesample/scenario/activity/LargeDataActivity.kt

```diff
@@ -34,9 +42,25 @@ class LargeDataActivity : BridgeBaseActivity() {
     }
 
     companion object {
-        fun getNavigationIntent(context: Context) = Intent(
-                context,
-                LargeDataActivity::class.java
-        )
+        private const val ARGUMENTS_KEY = "arguments"
+
+        fun getArguments(
+            activity: LargeDataActivity
+        ): LargeDataActivityArguments = activity
+                .intent
+                .getParcelableExtra(ARGUMENTS_KEY)
+
+        fun getNavigationIntent(
+            context: Context,
+            arguments: LargeDataActivityArguments
+        ) = Intent(context, LargeDataActivity::class.java
+        ).apply {
```

Comment:
The `).apply {` should be on the line above:

```kotlin
        fun getNavigationIntent(
            context: Context,
            arguments: LargeDataActivityArguments
        ) = Intent(context, LargeDataActivity::class.java).apply {
            ...
        }
```

Reply:
Done.